kubectl-pg/README.md

```diff
@@ -0,0 +1,64 @@
+# kubectl-pg
+
+### Installtion of kubectl pg plugin
+
+This project uses Go Modules for dependency management to build locally
+Install go and enable go modules ```export GO111MODULE=on```
+
+As this project doesn't contain vendor directory. 
+
+Generate vendor directory using ```go mod vendor```
```

Comment:
in what case a `vendor` directory may be needed for this project ?

Reply:
When u want to build this project locally all the imports will not be recognised so go mod vendor will create a vendor directory. All the imports will be fetched from that directory.